package.json

```diff
@@ -6,8 +6,7 @@
   "scripts": {
     "test": "npm run single && npm run local && npm run parallel",
```

Comment:
I think `npm run local` should be removed since we've deleted local test related files..
Also, why have we removed local tests btw?